### PR TITLE
fixes Issue 974

### PR DESCRIPTION
**Pull request by ThomasWaldmann**

### CODE REVIEW

File: borg/helpers.py [hunk 1]
```diff
@@ -658,7 +662,7 @@ def format_archive(archive):
     return '%-36s %s [%s]' % (
         archive.name,
         format_time(to_localtime(archive.ts)),
-        hexlify(archive.id).decode('ascii'),
+        bin_to_hex(archive.id),
```
Comment (enkore) [inline borg/helpers.py]: archive.fpr ?

File: borg/remote.py [hunk 2]
```diff
@@ -191,6 +191,10 @@ def __exit__(self, exc_type, exc_val, exc_tb):
             self.rollback()
         self.close()
 
+    @property
+    def id_str(self):
+        return bin_to_hex(self.id)
```
Comment (enkore) [inline borg/remote.py]: Hmmm, maybe just `__str__`? This ID is the only kind of string representation that would make sense for Manifest (and Repository)

Comment (ThomasWaldmann) [inline borg/remote.py]: __str__ is used at some places for a much more verbose representation that is used in screen output. archive and stats come to mind.

File: borg/helpers.py (hunk 1 above)
Comment (ThomasWaldmann) [inline borg/helpers.py]: i first had that and it failed - because "archive" is in reality an ArchiveInfo instance.

**Comment by ThomasWaldmann** [comment]
merge?

**Pull request merged**